.mocharc.js

```diff
@@ -20,4 +20,7 @@ module.exports = {
     '**/build/**',
     'docs/.next/**',
   ],
+  // detect-modules doesn't work with @babel/register
+  // https://github.com/babel/babel/issues/6737
+  'node-option': ['no-experimental-detect-module'],
```

Comment:
Cherry-picked from [`0f92cc3` (#45630)](https://github.com/mui/material-ui/pull/45630/commits/0f92cc363248f332ddb8fed2c6b424870b70d41e)

Reply:
This breaks again in Node 23, but ok, let's worry about that later